src/main/java/com/znsio/e2e/runner/Setup.java

```diff
@@ -232,7 +230,7 @@ public List<String> getExecutionArguments() {
 
         setupDirectories();
         setLogPropertiesFile();
-        setBrowserConfigFile();
+        setBrowserConfigFilePath();
```

Comment:
what would happen if someone is not used web at all? So no browser config required

Reply:
moved this method to setupWebExecution method so that we make sure it is used ONLY for web execution

Reply:
we'll give it a run tomorrow and see if this works as expected